code/src/App.js

```diff
@@ -1,9 +1,69 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+import { PopularList } from 'components/PopularList';
+import { Details } from 'components/Details';
+import NotFound from 'components/NotFound';
+import styled from 'styled-components'
+import LoadingPage from 'components/LoadingPage';
+
+const API_KEY = '3bf2860c1d44881bc87143482ddf0f94';
```

Comment:
Maybe its me, but i tried out the api key but didn't succeed. Should it led to some where ?